emark/views.py

```diff
@@ -93,3 +97,67 @@ def get(self, request, *args, **kwargs):
                 "Cache-Control": "no-cache, no-store, must-revalidate",
             },
         )
+
+
+class DashboardView(LoginRequiredMixin, UserPassesTestMixin, TemplateView):
+    """Show a dashboard of available email classes."""
+
+    template_name = "emark/dashboard.html"
+
+    def test_func(self):
+        return self.request.user.is_staff
```

Comment:
That isn't a very concrete permission limitation. If it's just the template without data, this should be ok though.

Reply:
I am planning on adding sending functionality later.

How would you see a more concrete permission management here? Allowing users to set a default permission class via settings?